dart/example_web/web/index.html

```diff
@@ -2,6 +2,21 @@
 
 <html>
 <head>
+    <!--
+      If you are serving your web app in a path other than the root, change the
+      href value below to reflect the base path you are serving from.
+
+      The path provided below has to start and end with a slash "/" in order for
+      it to work correctly.
+
+      For more details:
+      * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base
+
+      This is a placeholder for base href that will be replaced by the value of
+      the `--base-href` argument provided to `flutter build`.
+    -->
+    <base href="$FLUTTER_BASE_HREF">
```

Comment:
Default is just `/` https://github.com/bluefireteam/flutter-gh-pages/blob/57815b17b371455ec1a98f075b71b4c6ba0a938c/action.yml#L21-L24